src/components/Columns/Columns.tsx

```diff
@@ -1,13 +1,15 @@
 import { Grid, GridProps } from '@aws-amplify/ui-react';
 import classNames from 'classnames';
 
-interface ColumnsProps extends GridProps {
+interface ColumnsProps extends Omit<GridProps, 'as'> {
   children?: React.ReactNode;
   columns?: 2 | 3 | 4;
   size?: 'small' | 'medium';
+  as?: 'div' | 'ul';
```

Comment:
We need our grid container to be used as a `<ul>` so I added the "as" prop that we can pass to the underlying Grid element.